src/index.test.js

```diff
@@ -290,3 +290,19 @@ test('adds aria-label when having label on search input', t => {
   t.deepEqual(wrapper.find('.search').prop('aria-labelledby'), undefined)
   t.deepEqual(wrapper.find('.search').prop('aria-label'), 'hello world')
 })
+
+test('appends selected tags to aria-labelledby with provided aria-labelledby', t => {
+  const { tree } = t.context
+  tree[0].checked = true
+  const wrapper = mount(<DropdownTreeSelect id="rdts" data={tree} texts={{ label: '#hello #world' }} />)
+  t.deepEqual(wrapper.find('.dropdown-trigger').prop('aria-labelledby'), 'hello world rdts-0_tag')
+  t.deepEqual(wrapper.find('.dropdown-trigger').prop('aria-label'), undefined)
+})
+
+test('appends selected tags to aria-labelledby with text label', t => {
+  const { tree } = t.context
+  tree[0].checked = true
+  const wrapper = mount(<DropdownTreeSelect id="rdts" data={tree} texts={{ label: 'hello world' }} />)
+  t.deepEqual(wrapper.find('.dropdown-trigger').prop('aria-labelledby'), 'rdts_trigger rdts-0_tag')
+  t.deepEqual(wrapper.find('.dropdown-trigger').prop('aria-label'), 'hello world')
```

Comment:
We don’t wanna rely on snapshots for these checks (this and above one)?

Reply:
Do you want me to switch to snapshots instead?

Reply:
Explicit testing is OK for a handful of assets. You can also express the intent by using inline snapshots but I’m not sure if Ava supports them yet. Let’s leave these as is for now.